pyvista/plotting/plotting.py

```diff
@@ -3500,35 +3516,24 @@ def add_volume(
 
         self.mapper.scalar_range = clim
 
-        # Set colormap and build lookup table
-        table = _vtk.vtkLookupTable()
-        # table.SetNanColor(nan_color) # NaN's are chopped out with current implementation
-        # above/below colors not supported with volume rendering
-
-        if isinstance(annotations, dict):
-            for val, anno in annotations.items():
-                table.SetAnnotation(float(val), str(anno))
+        if not has_module('matplotlib'):
+            raise ImportError('Please install matplotlib for volume rendering.')
```

Comment:
We can't use just a LUT with volume rendering??

Reply:
Non-trivial, but it was implemented and had the side effect of pulling more logic out of `pyvista.Plotter`.